packages/slider/test/slider.test.ts

```diff
@@ -1604,4 +1603,75 @@ describe('Slider', () => {
         await elementUpdated(el);
         expect(el.values).to.deep.equal({ a: 10, b: 20, c: 29 });
     });
+    it('resets to default value on double click after moving pointer', async () => {
```

Comment:
We should add a test where the Slider is in an Overlay to confirm it does not close the Overlay.

We should also add `default-value` to one or some of the Stories in Overlay to confirm this functionality there, as well.

Reply:
Ping.